report-qualification-template.R

```diff
@@ -0,0 +1,153 @@
+#' @title report-qualification-template
+#' @description
+#' Function that evaluates a model and generate its Qualification Report
+#' for `Open-Systems-Pharmacology/` sub-repository
+#' @param modelIndex Index of model to run
+#' @param modelsData data.frame defining input data of models
+#' @param toolsData data.frame defining tools used to evaluate models and generate reports
+runEvaluationReport <- function(modelIndex, modelsData, toolsData) {
+  if (!modelsData$Execute[modelIndex]) {
+    return()
+  }
+  library(ospsuite.reportingengine)
+  ospsuite::clearMemory(clearSimulationsCache = TRUE)
+
+  qualificationProject <- modelsData$`Repository name`[modelIndex]
+  modelName <- modelsData$`Snapshot name`[modelIndex]
+  snapshotFile <- paste0(modelName, ".json")
+  workingDirectory <- normalizePath(qualificationProject, mustWork = FALSE, winslash = "/")
+  qualificationRunnerFolder <- "QualificationRunner/QualificationRunner"
+  pkSimPortableFolder <- "PK-Sim/PK-Sim"
+  versionInfo <- QualificationVersionInfo$new(
+    "1.0",
+    toolsData$Version[toolsData$Tool %in% "OSP Suite"],

```

Comment:
This should be the "major.minor" version of **PK-Sim** (e.g. "11.3" for PK-Sim version "11.3.456")